### PR TITLE
Adding note that you must manually create Ingress DNS records if prox…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…y is enabled

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@danehans can you please review this update? I put the note both in the proxy docs, and in the section on creating the Ingress DNS records.

For example, here is the preview for the AWS UPI docs:

* https://proxy-dns--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-configure-proxy_installing-aws-user-infra
* https://proxy-dns--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-create-ingress-dns-records_installing-aws-user-infra

File: modules/installation-create-ingress-dns-records.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [id="installation-create-ingress-dns-records_{context}"]
 = Creating the Ingress DNS Records
 
-If you removed the DNS Zone configuration, manually create DNS records that point to the Ingress load balancer.
+If you removed the DNS Zone configuration, or if you enabled the cluster-wide proxy, manually create DNS records that point to the Ingress load balancer.
```
Comment (danehans) [inline modules/installation-create-ingress-dns-records.adoc]: s/Ingress load balancer/hostname of the router load balancer service/. You may want to provide an example on how to get the Ingress load-balancer. Here's an example for the default router:
```
$ oc get svc/router-default -n openshift-ingress -o yaml | grep hostname
    - hostname: a4a37c40e0af011eabdc302eee56a296-792674788.us-west-2.elb.amazonaws.com
```

File: modules/installation-configure-proxy.adoc [hunk 2]
```diff
@@ -94,6 +103,22 @@ Only the Proxy object named `cluster` is supported, and no additional
 proxies can be created.
 ====
 
+ifdef::aws-upi,aws-restricted-upi,gcp-upi[]
+
+When the cluster-wide proxy is enabled, you must manually create the Ingress DNS records.
+
+// TODO: xref: https://docs.openshift.com/container-platform/4.2/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-create-ingress-dns-records_installing-aws-user-infra
```
Comment (danehans) [inline modules/installation-configure-proxy.adoc]: If the TODO stays, please add some context for others to understand.

File: modules/installation-create-ingress-dns-records.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/installation-create-ingress-dns-records.adoc]: @danehans What would someone need to know the host name for? (I don't see anything in this procedure that uses it).

File: modules/installation-configure-proxy.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/installation-configure-proxy.adoc]: Unfortunately we can't currently link between modules. This is just a placeholder if we ever can in the future, and I would use this link in the above sentence.

File: modules/installation-configure-proxy.adoc [hunk 3]
```diff
@@ -94,6 +103,22 @@ Only the Proxy object named `cluster` is supported, and no additional
 proxies can be created.
 ====
 
+ifdef::aws-upi,aws-restricted-upi,gcp-upi[]
```
Comment (kalexand-rh) [inline modules/installation-configure-proxy.adoc]: If you expect that the same statement will always be the same for all UPI cloud clusters, please use the same context key for each cloud. It'll make it slightly easier to add and maintain other UPI clouds, and I know that several are in the queue.

File: modules/installation-create-ingress-dns-records.adoc [hunk 4]
```diff
@@ -6,7 +6,7 @@
 [id="installation-create-ingress-dns-records_{context}"]
 = Creating the Ingress DNS Records
 
-If you removed the DNS Zone configuration, manually create DNS records that point to the Ingress load balancer.
+If you removed the DNS Zone configuration, or if you enabled the cluster-wide proxy, manually create DNS records that point to the host name of the router load balancer service.
```
Comment (kalexand-rh) [inline modules/installation-create-ingress-dns-records.adoc]: I'd say "If you removed the DNS Zone configuration or enabled the cluster-wide proxy," to place equal importance on both options.

File: modules/installation-gcp-user-infra-adding-ingress.adoc [hunk 5]
```diff
@@ -6,8 +6,8 @@
 = Optional: Adding the ingress DNS records
 
 If you removed the DNS Zone configuration when creating Kubernetes manifests and
-generating Ignition configs, you must manually create DNS records that point at
-the ingress load balancer. You can create either a wildcard
+generating Ignition configs, or if you enabled the cluster-wide proxy, you must manually create DNS records that point to
```
Comment (kalexand-rh) [inline modules/installation-gcp-user-infra-adding-ingress.adoc]: Same here.

**Review by kalexand-rh — COMMENTED** [review]
I have a request and a suggestion, but this LGTM.

**Comment by vikram-redhat** [comment]
@bergerhoffer can this be updated or closed if no longer required?

**Pull request closed without merging**